src/app/clients/add/formSections/babyProductCard.tsx

```diff
@@ -0,0 +1,75 @@
+import RadioGroupInput from "@/components/DataInput/RadioGroupInput";
+import {
+    Error,
+    errorExists,
+    ErrorSetter,
+    errorText,
+    Fields,
+    FieldSetter,
+    FormErrors,
+    onChange,
+} from "@/components/Form/formFunctions";
+import FreeFormTextInput from "@/components/DataInput/FreeFormTextInput";
+import React from "react";
+import GenericFormCard from "@/components/Form/genericFormCard";
+import { selectChangeEventHandler } from "@/components/DataInput/inputHandlerFactories";
+
+interface CardProps {
+    formErrors: FormErrors;
+    errorSetter: ErrorSetter;
+    fieldSetter: FieldSetter;
+    fields: Fields;
+}
+
```

Comment:
I noticed that CardProps seems to be used among all the formSections except with a bit of variation - would it be more efficient if you define one and import it to all these formSections?

```
interface CardProps {
    formErrors?: FormErrors;
    errorSetter?: ErrorSetter;
    fieldSetter: FieldSetter;
    fields?: Fields;
}
```

Reply:
Yep, I've considered that but I thought that if we make certain fields optional, the necessary props might not be passed in for a specific card. 
Alternatively, we can make all of them required and just pass them in but not use them - but right now eslint doesn't allow unused variables.

Reply:
Have now implemented what you suggested ! 😄